concent-builder/configure.yml

```diff
@@ -0,0 +1,80 @@
+- hosts: all
+  tasks:
+    - become:      yes
+      become_user: root
+      block:
+
+        - name: Install required packages for apt_key and apt_repository
+          apt:  name={{ item }}
+          with_items:
+            - apt-transport-https
+            - ca-certificates
+            - curl
+
+        - name: Add Google SDK repository_key
+          apt_key:
+            url:   https://packages.cloud.google.com/apt/doc/apt-key.gpg
+            state: present
+
+        - name: Add Google Cloud SDK repository
+          apt_repository:
+            repo:  "deb http://packages.cloud.google.com/apt cloud-sdk-{{ ansible_distribution_release }} main"
+            state: present
+
+        - name: Add Docker repository_key
+          apt_key:
+            url:   https://apt.dockerproject.org/gpg
+            state: present
+
+        - name: Add Docker.io apt repository
+          apt_repository:
+            repo:  "deb https://apt.dockerproject.org/repo {{ ansible_distribution|lower }}-{{ ansible_distribution_release }} main"
+            state: present
+
+        - name: Install system updates for Debian
+          apt:  update_cache=yes
+
+        - name: Install basic packages
+          apt:  name={{ item }}
+          state: present
+          with_items:
+            - ncdu
+            - tree
+            - htop
+            - tmux
+
+        - name: Install dependencies
+          apt:  name={{ item }}
+          with_items:
+            - build-essential
+            - python2.7
+            - python3
+            - python-pip
+            - git
+            - kubectl
+            - docker-engine
+            - google-cloud-sdk
+            - ansible
+            - virtualenv
+
+        - name:    apt autoclean
+          command: apt autoclean
+        - name:    apt autoremove
+          command: apt autoremove
+
+        - name: Make a Docker group
```

Comment:
Creating the `docker` group is not necessary. It should be created by docker automatically.